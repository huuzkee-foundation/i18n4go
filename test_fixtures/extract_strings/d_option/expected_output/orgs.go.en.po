# filename: d_option/input_files/org/orgs.go, offset: 755, line: 28, column: 16
msgid "orgs"
msgstr "orgs"

# filename: d_option/input_files/org/orgs.go, offset: 778, line: 29, column: 16
msgid "o"
msgstr "o"

# filename: d_option/input_files/org/orgs.go, offset: 798, line: 30, column: 16
msgid "List all orgs"
msgstr "List all orgs"

# filename: d_option/input_files/org/orgs.go, offset: 830, line: 31, column: 16
msgid "CF_NAME orgs"
msgstr "CF_NAME orgs"

# filename: d_option/input_files/org/orgs.go, offset: 1142, line: 43, column: 13
msgid "Getting orgs as %s...\n"
msgstr "Getting orgs as %s...\n"

# filename: d_option/input_files/org/orgs.go, offset: 1267, line: 46, column: 33
msgid "name"
msgstr "name"

# filename: d_option/input_files/org/orgs.go, offset: 1465, line: 56, column: 17
msgid "Failed fetching orgs.\n%s"
msgstr "Failed fetching orgs.\n%s"

# filename: d_option/input_files/org/orgs.go, offset: 1541, line: 61, column: 14
msgid "No orgs found"
msgstr "No orgs found"

