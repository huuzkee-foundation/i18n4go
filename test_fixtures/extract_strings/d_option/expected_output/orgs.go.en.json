[{"id":"orgs","translation":"orgs"},{"id":"o","translation":"o"},{"id":"List all orgs","translation":"List all orgs"},{"id":"CF_NAME orgs","translation":"CF_NAME orgs"},{"id":"Getting orgs as %s...\n","translation":"Getting orgs as %s...\n"},{"id":"name","translation":"name"},{"id":"Failed fetching orgs.\n%s","translation":"Failed fetching orgs.\n%s"},{"id":"No orgs found","translation":"No orgs found"}]
